basic/src/App.tsx

```diff
@@ -0,0 +1,12 @@
+import { Outlet } from "react-router-dom";
+import "./App.css";
+import { Header } from "./shared/components/Header";
+
+export function App() {
+  return (
+    <main className="flex min-h-screen w-full flex-col bg-neutral-200 text-black">
```

Comment:
```suggestion
    <main className="flex min-h-screen w-full flex-col bg-neutral-50">
```
Still rooting for the lighter gray bg color. Also, they say ["never use block for text color"](https://uxplanet.org/basicdesign-never-use-pure-black-in-typography-36138a3327a6) so I'd drop the `text-black` and use something like `text-gray-800`.